kedro_kubeflow/config.py

```diff
@@ -0,0 +1,101 @@
+from kedro.config import MissingConfigException
+
+DEFAULT_CONFIG_TEMPLATE = """
+
+host: {url}
+
+run_config:
+  image: {image}
+  image_pull_policy: IfNotPresent
+  experiment_name: {project}
+  run_name: {project}
+  wait_for_completion: False
+  volume:
+    storageclass: # default
+    #size: 1Gi
+    #access_modes: [ReadWriteOnce]
+"""
+
+
+class Config(object):
+    def __init__(self, raw):
+        self._raw = raw
+
+    def _get_or_default(self, prop, default):
+        return self._raw[prop] if prop in self._raw.keys() else default
+
+    def _get_or_fail(self, prop):
+        if prop in self._raw.keys():
+            return self._raw[prop]
+        else:
+            raise MissingConfigException(
+                f"Missing required configuration: '{self._get_prefix()}{prop}'."
+            )
+
+    def _get_prefix(self):
+        return ""
+
+    def __eq__(self, other):
+        return self._raw == other._raw
+
+
+class VolumeConfig(Config):
+    @property
+    def storageclass(self):
+        return self._get_or_fail("storageclass")
+
+    @property
+    def size(self):
+        return self._get_or_default("size", "1Gi")
+
+    @property
+    def access_modes(self):
+        return self._get_or_default("access_modes", "[ReadWriteOnce]")
```

Comment:
Let's make ReadWriteMany as a default, to allow multiple steps to use the volume in parall